src/main/java/Tickles.java

```diff
@@ -0,0 +1,125 @@
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+public class Tickles {
+
+    private static boolean stillGoing = true;
+    private static int totalTasks = 0;
+    private static List<Task> list = new ArrayList<>();
+    public static void main(String[] args) {
+        String logo =
+            "╱╭╮╱╱╱╱╭╮╱╭╮\n" +
+            "╭╯╰╮╱╱╱┃┃╱┃┃\n" +
+            "╰╮╭╋┳━━┫┃╭┫┃╭━━┳━━╮\n" +
+            "╱┃┃┣┫╭━┫╰╯┫┃┃┃━┫━━┫\n" +
+            "╱┃╰┫┃╰━┫╭╮┫╰┫┃━╋━━┃\n" +
+            "╱╰━┻┻━━┻╯╰┻━┻━━┻━━╯\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println(" What can I do for you?");
+        while (stillGoing) {
+            promptUser();
+        }
+    }
+
+    // Prompts user for input, and handles the special input cases "bye", "unmark"/"mark", "list".
+    public static void promptUser() {
+        Scanner in = new Scanner(System.in);
+        String prompt = in.nextLine();
+
+//        switch (prompt) {
+//        case "bye":
+//            stillGoing = false;
+//            printThis("Bye. Hope to see you again soon! Mr. Tickles will miss you.");
+//        case "list":
+//            printList();
+//        }
+
+        if (prompt.equals("bye")) {
+            stillGoing = false;
+            printThis("Bye. Hope to see you again soon! Mr. Tickles will miss you.");
+        } else if (prompt.equals("list")) {
+            printList();
+        } else if (prompt.length() == 8 && prompt.substring(0,6).equals("unmark")) {
```

Comment:
You can check if the string contains the word unmark by,
`prompt.contains("unmark")`